whisper/whisperv6/filter_test.go

```diff
@@ -315,11 +314,13 @@ func TestMatchEnvelope(t *testing.T) {
 		t.Fatalf("failed Wrap with seed %d: %s.", seed, err)
 	}
 	match := fsym.MatchEnvelope(env)
-	if match {
+	if !match {
+		// topic mismatch should have no affect, as topics are handled by topic matchers
```

Comment:
no effect

Reply:
It seems to me that you are effectively disabling your unit test here; why not remove that part altogether?